src/main/java/com/mercadolibre/be_java_hisp_w24_g02/dao/CreatePostDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.mercadolibre.be_java_hisp_w24_g02.dao;
+
+import jakarta.validation.constraints.NotNull;
+
+public record CreatePostDTO(
+        @NotNull
+        Integer user_id,
```

Comment:
Try to use cammelCase to name atributtes, you can rename the key of json using @JsonProperty anotation, same to the other atributtes in CreateProductDTO